### PR TITLE
update calico-policies

### PR DESCRIPTION
**Pull request by sanchezma5**
- remove decommissioned IPs
- add policies for eu-es

### CODE REVIEW

**Review by bradbehle — CHANGES_REQUESTED** [review]

File: calico-policies/private-network-isolation/au-syd/allow-egress-pods-private.yaml
Comment (sanchezma5) [reply calico-policies/private-network-isolation/au-syd/allow-egress-pods-private.yaml]: proxy CSE addresses have been restored

**Review by bradbehle — APPROVED** [review]

File: calico-policies/private-network-isolation/eu-es/allow-egress-pods-private.yaml
Comment (ctrath) [inline calico-policies/private-network-isolation/eu-es/allow-egress-pods-private.yaml:57]: We should probably change this comment.  We are allowing pod egress to private networks here, not denying

**Review by ctrath — APPROVED** [review]

File: calico-policies/private-network-isolation/eu-es/allow-egress-pods-private.yaml
Comment (bradbehle) [reply calico-policies/private-network-isolation/eu-es/allow-egress-pods-private.yaml:57]: Agreed, let's fix this in a separate PR

**Comment by bradbehle** [comment]
Since this is time sensitive, I'm going to override our rule of requiring 2 approvals from @kersten1 @derekpoindexter and myself (code owners for these files) since I don't see them online, and also since @ctrath approved as well, and merge this.

**Pull request merged**
